cmd/maya-apiserver/cstor-operator/cspc/handler.go

```diff
@@ -182,6 +192,11 @@ func (c *Controller) syncCSPC(cspc *apis.CStorPoolCluster) error {
 		pc.createDeployForCSPList(cspList)
 	}
 
+	if pendingPoolCount == 0 {
```

Comment:
does it makes sense to continue, i.e., reach till here even if `pc.create` or `GetCSPWithoutDeployment` or `pc.DownScalePool` returns error?
Basically, my qn is: even if one particular day-2 op fails, we can try for other operations that user asked for

Reply:
oops.. I will put this comment in other open PR.. here, in this PR, you can ignore this comment